packages/amplify-cli-core/src/types.ts

```diff
@@ -201,7 +391,15 @@ export type GetPackageAssetPaths = () => Promise<string[]>;
 /**
  * Placeholder type
  */
-export type $IPluginManifest = $TSAny;
+export type $IPluginManifest = {
```

Comment:
This is neither interface nor implements interface, why is it prefixed with `$` and `I` ?